docs/pql-guide/pql-data-types.md

```diff
@@ -0,0 +1,53 @@
+---
+title: PQL Options
```

Comment:
These aren't PQL options or PQL types, these are specific to some of the IDK configs. I think this page needs to be moved out of PQL and into com ingest if needed

Reply:
I'll shift the content into a temp include file so I know where it is and then review later with @jrbrinlee1

Reply:
currently added to /_includes/pql-guide as ingest-pql-data-types